### ISSUE TITLE
Recirculation feature no longer working?

### ISSUE THREAD

**Issue by guillochon** [dustindclark/homebridge-rinnai-controlr#9]
**Describe The Bug:**
Can no longer turn recirculation on with the plug-in. The state of recirculation still appears to be read correctly (if I turn it on externally via the control-r app), but I can no longer turn on recirc via this plugin.

**To Reproduce:**
Try to turn on recirculation via the homebridge switch, nothing happens.

**Expected behavior:**
Recirculation should turn on.

**Logs:**
No logs produced.

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by Bleh55** [comment]
I just noticed this same thing last night..

**Comment by tedinorbit** [comment]
I’m noticing this as well. I also cannot change temperature. Not sure if it’s related

**Comment by tedinorbit** [comment]
I just noticed a firmware update on my control-r unit was available. Perhaps there were some breaking api changes?

**Comment by Bleh55** [comment]
No mention of what the fix is, but the Home Assistant forum seems to have identified same/similar issue, and may have a fix (last few comments). 

https://community.home-assistant.io/t/rinnai-tankless-water-heater-control-r-module-integration/100646/38

**Comment by explosivo22** [comment]
Yeah Rinnai made a part of the API that was being utilized forbidden and changed it over to an authenticated request only. I hope everything is fixed and working as of now, but doesn't mean more changes are coming or a bug managed to be introduced.

**Comment by guillochon** [comment]
@dustindclark do you think you'd be able to apply the fix above to the homebridge plugin?

**Comment by guillochon** [comment]
I can verify that this fix works for me! Thanks so much @dustindclark!
